src/pages/CreatePool/index.tsx

```diff
@@ -112,32 +137,56 @@ export default function CreatePool(): React.ReactElement {
 
   const poolNameError = poolName.length > 10
```

Comment:
I feel like the name will be longer than the symbol almost every time - I think this limit should be the max length of bytes32